index.html

```diff
@@ -32,20 +33,28 @@
     </script>
   </head>
   <body>
+    <div class="menu-space">
+      <div class="fa fa-close" id="close_icon"></div>
+      <div class="menu_links">
+        <a href="#intro" class="menu_link">Portfolio</a>
+        <a href="#about" class="menu_link">About</a>
+        <a href="#contact" class="menu_link">Contact</a>
+      </div>
+    </div>
     <header>
       <div class="logo"><a href="#">My logo</a></div>
       <nav>
         <div class="fa fa-bars" id="menu"></div>
```

Comment:
- [ ] Please make sure that the size of `hamburger` icon is the same as provided in Figma. Currently, the button on your project looks too small:

Yours | Figma
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/150954338-c020675c-8354-45ec-9bb0-fbe7fdef81dc.png)  | ![image](https://user-images.githubusercontent.com/77038610/150954300-863da915-9eae-4757-9b94-2e7cb2a281ec.png)